src/behaviors/focusZone.ts

```diff
@@ -343,9 +343,18 @@ export function focusZone(container: HTMLElement, settings?: FocusZoneSettings):
   const activeDescendantControl = settings?.activeDescendantControl
   const activeDescendantCallback = settings?.onActiveDescendantChanged
 
-  function updateTabIndex(from?: HTMLElement, to?: HTMLElement) {
+  let activeDescendantSuspended = Boolean(activeDescendantControl)
+  let currentFocusedElement = activeDescendantControl ? undefined : focusableElements[0]
```

Comment:
Since `focusableElements` is set to `[]` on line 336, isn’t `focusableElements[0]` also `undefined`?

```suggestion
  let currentFocusedElement: HTMLElement | undefined
```